### ISSUE TITLE
Do not autoequip stage weapons when selecting stage

### ISSUE THREAD

**Issue by luismartinezs** [Danmer/vs-build-planner#65]
If I click on a stage with weapons (e.g. Astral), these weapons are automatically added to the equipped weapons, instead they should be listed in the same place the passives are listed (not equipped, just dispalyed under the character squares)

**Comment by wraith309** [comment]
Agreed, sometimes I'll try making a character, and if i select the stage first then i will need to remove and re add the weapon because the build planner will imply that the stage weapon would take up one of my slots when in reality i could just wait until I've filled the slots before picking it up.

Putting the stage weapon in the extra space would more accurately reflect how the weapon is obtainable past your limit, the same way stage items are displayed

**Comment by Danmer** [comment]
I wasn't sure and you're probably right. But before we do it, I would like to resolve three potential problems:
- there is no way to exclude stage items and show evolved weapons in the extra slots for now
- stage weapons should be correctly handled by our favorite Silent Old Sanctuary
- stage weapons should be correctly displayed if they are starting weapons or a part of evolved starting weapons

Thanks for the suggestion anyway.

**Comment by wraith309** [comment]
damn, you're right those are some challenging structural issues that would preclude placing stage weapons down there.

would it be possible to instead give the (non starting weapon) stage weapons priority for being placed on the right with the extra weapons when you exceed 6 weapon held? or does that also run into some of the same issues?
